Automattic/jetpack
# Boost: Dismiss the admin notice when the plugin is deactivated

#### Changes proposed in this Pull Request:

Fixes e2e test error introduced in #25980


#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Apply the patch, change the theme, make sure the regeneration notice appears and dismiss it.
